Assets/JavaScript/booksList-3.js

```diff
@@ -0,0 +1,98 @@
+class AwsomeBooks {
+  booksListArr;
+
+  constructor(booksListArr) {
+    this.booksListArr = booksListArr;
+    this.id = booksListArr.length - 1;
+    this.newBook = document.querySelector('.booksList');
+    this.displayBooks();
+    this.addButton = document.querySelector('#add-button');
+    this.addButton.addEventListener('click', () => this.addBook());
+
+    this.removeButtons = document.querySelectorAll('.remove-button');
+
+    for (let j = 0; j < this.removeButtons.length; j += 1) {
+      this.removeButtons[j].addEventListener('click', () => this.removeBook(this.removeButtons[j].id));
+    }
+  }
+
+  changeColor() {
+    for (let i = 0; i < this.booksListArr.length; i += 1) {
+      const bgColor = document.querySelector(`.item-${this.booksListArr[i].id}`);
+      if (i % 2 === 0) {
+        bgColor.style.backgroundColor = 'gray';
+      } else {
+        bgColor.style.backgroundColor = 'white';
+      }
+    }
+  }
+
+  displayBooks() {
+    this.newBook.innerHTML = this.booksListArr
+      .map(
+        (x) => `
+            <div class="item-${x.id}">
+            <p>Title: ${x.title}</p>
+            <p>Author:${x.author}</p>
+            <input type = "button" class = "remove-button" id='button-item-${x.id}' value="remove">        
+            </div>
+            `,
+      )
+      .join('');
+    this.changeColor();
+  }
```

Comment:
![Capture](https://user-images.githubusercontent.com/93039359/211899672-3857b6bb-7c4a-4106-8659-8ec117276d97.PNG)

- [ ] It's great that you have fixed the error suggested by the previous reviewer :+1: but i guess you broke something on the other hand. When i start your application i can see an existing book with undefined title and author. When i try to remove it then later after refreshing the page it comes back :thinking: Kindly fix this :+1: